upload/src/addons/SV/WarningImprovements/XF/Admin/Controller/Warning.php

```diff
@@ -17,25 +17,43 @@ public function actionIndex(ParameterBag $params)
         if ($response instanceof \XF\Mvc\Reply\View)
         {
             $categoryRepo = $this->getCategoryRepo();
-            $categories = $categoryRepo->findCategoryList()->fetch();
+            $categories = $categoryRepo->findCategoryList()
+                ->where('warning_count', '<>', 0)
```

Comment:
Need to be able to see empty categories, or kinda stuck on how to add warnings to them.